src/firefly/js/visualize/ui/VisToolbarView.jsx

```diff
@@ -155,10 +155,19 @@ VisToolbarViewWrapper.propTypes= {
 export class VisToolbarView extends PureComponent {
     constructor(props) {
         super(props);
+
+        this.selectCallBack = this.selectCallBack.bind(this);
+        const dl = getDrawLayerByType(getDlAry(), SelectArea.TYPE_ID);
```

Comment:
VisToolbarView is not suppose to have any state.  The class above VisToolbar does keep state.  However, that does not work either because there is the case where I delete the select drawing layer from the LayerControl menu.  I that case there is a bug, because the toolbar show selection is on but it is not.  The toolbar does not go back to selection off.